src/modules/maps/containers/MapVisuals.jsx

```diff
@@ -7,8 +7,8 @@ import { MapHelper } from '../../../helpers/mapexplorer.js';
 import { enableViewCameraMode } from '../actions/mapexplorer';
 
 const mapconfig = require('../../../constants/mapExplorer.config.json');
-const gorongosaGeodata = require('../../../map-data/gorongosa-geodata.json');
-const vegetationGeodata = require('../../../map-data/vegetation-geodata.json');
+const gorongosaGeodata = require('../data/gorongosa-geodata.json');
+const vegetationGeodata = require('../data/vegetation-geodata.json');
```

Comment:
The path should be changed here as well https://github.com/zooniverse/wildcam-gorongosa-education/blob/tutorial-dialogs/src/modules/maps/components/MapLegendVegetation.jsx#L2